en/docs/learn/enterprise-integration-patterns/messaging-systems/message-endpoint.md

```diff
@@ -1,83 +1,93 @@
 # Message Endpoint
 
+This page explains how you can implement a sample scenario of Message Endpoint EIP using WSO2 Micro Integrator.
+
+## Introduction to Message Endpoint EIP
+
 The Message Endpoint EIP encapsulates the messaging system from the rest of the application and customizes a general messaging API for a specific application and task. Therefore, you can change the message API just by changing the endpoint code. This improves the maintainability of applications.
 
 !!! info 
     For more information, see the [Message Endpoint](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageEndpoint.html) documentation.
 
 ## Sample scenario
 
-The example scenario depicts how a stock quote is generated when a request is sent to the ESB profile of WSO2 EI. The sender sends the request to the ESB profile, where the request is then diverted to the Stock Quote service. 
+The example scenario depicts how a stock quote is generated when a request is sent to the WSO2 MI. The sender sends the request to the WSO2 MI, where the request is then diverted to the Stock Quote service. 
 
 ![Message endpoint pattern]({{base_path}}/assets/img/learn/enterprise-integration-patterns/messaging-systems/message-endpoint-pattern.png)
 
 ## Synapse configurations of the artifacts
 
 !!! note
-    When you unzip the ZIP file you download below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/synapse-config` directory. For more information about these artifacts, go to WSO2 EI Documentation.
-
-**Proxy Service**
-
-```
-<proxy name="message-endpoint-proxy" startOnLoad="true" transports="http https"
-    xmlns="http://ws.apache.org/ns/synapse">
-    <target>
-        <inSequence>
-            <!-- Sends the message to the specified service -->
-            <send>
-                <endpoint>
-                    <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
-                </endpoint>
-            </send>
-        </inSequence>
-        <outSequence>
-            <respond/>
-        </outSequence>
-        <faultSequence/>
-    </target>
-</proxy>
-```
+    When you unzip the ZIP file you downloaded below in step 7 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs

Reply:
let's point to https://mi.docs.wso2.com/en/latest/develop/intro-integration-development/